docs/src/main/asciidoc/logging.adoc

```diff
@@ -154,62 +203,84 @@ WARNING:: Same as **WARN**.
-opening the door to optimization opportunities.
-As an example, in native execution the minimum level enables lower level checks (e.g. `isTraceEnabled`) to be folded to `false`,
-resulting in dead code elimination for code that will never to be executed.
+The minimum logging level sets a floor level that Quarkus needs to generate, potentially opening the door to optimization opportunities.
+
+.An example from native execution:
+Setting the `INFO` as the minimum logging level sets lower-level checks, such as `isTraceEnabled`, to `false`.
+This identifies code like `if(logger.isDebug()) callMethod();` that will never be executed and mark it as "dead."
+
+If you add these properties by using the command line, ensure the `"` character is escaped.
+
+[WARNING]
+====
+If you add these properties by using the command line, ensure the `"` character is always escaped as shown below:
+----
+ `-Dquarkus.log.category.\"org.hibernate\".level=TRACE`
+----
+====
 
-All possible properties are listed in the <<loggingConfigurationReference,logging configuration reference>> section.
+All potential properties are listed in the <<loggingConfigurationReference,logging configuration reference>> section.
 
-NOTE: If you are adding these properties by using the command line, make sure `"` is escaped.
-For example `-Dquarkus.log.category.\"org.hibernate\".level=TRACE`.
 
 === Logging categories
 
-Logging is done on a per-category basis.
-Each category can be independently configured.
-A configuration which applies to a category will also apply to all sub-categories of that category,
-unless there is a more specific matching sub-category configuration.
-For every category the same settings that are configured on ( console / file / syslog ) apply.
-These can also be overridden by attaching one or more named handlers to a category.
-See example in the <<category-named-handlers-example>> output.
+Logging is done on a per-category basis, with each category being configured independently.
+A category configuration recursively applies to all subcategories of that category unless there is a more specific matching sub-category configuration.
+
+The parent of all logging categories is called the "root category".
+This category, being the ultimate parent, may contain configuration which applies globally to all other categories. This includes the globally configured handlers and formatters.
+
+Thus, configurations made under `quarkus.log.console.*`, `quarkus.log.file.*`, and `quarkus.log.syslog.*`, are global and apply for all categories. This is described in link:https://quarkus.io/guides/logging#configuration[].
+
+If you want to configure something extra for a specific category, create a named handler like `quarkus.log.handler.[console|file|syslog].<your-handler-name>.*` and set it up for that category by using `quarkus.log.category.<my-categories>.handlers`.
+
+//TODO: Add a better, real-world example of a handler configuration for a more specific category. CC DML
```

Comment:
The handler configuration is down below so *instead* of doing this here, I think that each handler down below (file, console, etc.) should have an example of global and per-category configuration.

Reply:
I agree. Please, if you can provide me these examples, I will add them there. Or would you rather do it in separate PR when this is merged?

Reply:
Great, I will star a PR for a minimalistic review of Logging guide includes. I think that will be great opportunity to add these examples.